float32 head_delta
float32 platform_delta
---
bool head_success
